js/mediaView.js

```diff
@@ -503,22 +502,23 @@ class MediaView extends ComponentView {
 
     $button.attr('aria-expanded', true);
     $buttonText.html(this.model.get('_transcript').inlineTranscriptCloseButton);
+    this.transcriptTriggers('opened');
   }
   
-  onExternalTranscriptClicked(event) {
-    this.transcriptTriggers();
+  onExternalTranscriptClicked() {
+    this.transcriptTriggers('external');
   }
   
-  transcriptTriggers() {
+  transcriptTriggers(state) {
     const setCompletionOnView = this.model.get('_transcript')._setCompletionOnView;
     const isComplete = this.model.get('_isComplete');
     const shouldComplete = (setCompletionOnView && !isComplete);
     
     if (!shouldComplete) {
-      return Adapt.trigger('media:transcript', this);
+      return Adapt.trigger('media:transcript', state, this);
     }
     this.setCompletionStatus();
-    Adapt.trigger('media:transcriptComplete', this);
+    Adapt.trigger('media:transcriptComplete', state, this);
```

Comment:
```suggestion
    Adapt.trigger('media:transcript', state, this);
    Adapt.trigger('media:transcript', 'complete', this);
```

There seems no point having a separate event now that there's a state component.